docs/nodle-wallets/nodle-cash/Hotspots.mdx

```diff
@@ -0,0 +1,48 @@
+---
+sidebar_position: 5
+---
+
+# Nodle Hotspots
+
+![Title](/img/docs/nodle-cash/HotSpots.png)
```

Comment:
Where is this image taken from? It was created for a different purpose. If we need to add an image here, then please provide dimensions, so that we can prepare an image.